app/internal/page/sms_sender.go

```diff
@@ -0,0 +1,31 @@
+package page
+
+import (
+	"context"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/notify"
+)
+
+type SMSSender struct {
+	notifyClient NotifyClient
+}
+
+func NewSMSSender(client NotifyClient) *SMSSender {
+	return &SMSSender{
+		notifyClient: client,
+	}
+}
+
+func (s *SMSSender) PaperCertificateProviderMeetingPrompt(ctx context.Context, lpa *Lpa, appData AppData, template notify.TemplateId) error {
+	_, err := s.notifyClient.Sms(ctx, notify.Sms{
```

Comment:
I think you could just inline this call, it isn't really providing anything extra

Reply:
True 👍